### PR TITLE
Update item descriptions

### PR DESCRIPTION
**Pull request by YaBoiJD**

### CODE REVIEW

File: data/items.js [hunk 1]
```diff
@@ -378,7 +378,7 @@ let BattleItems = {
 		num: 851,
 		gen: 7,
 		isPokeball: true,
-		desc: "A special Poke Ball designed to catch Ultra Beasts.",
+		desc: "A Poké Ball that has a low success rate for catching a Pokémon.",
```
Comment (Zarel) [inline data/items.js]: Descriptions are supposed to be clear and useful, you're not supposed to just copy the in-game description.

Comment (Zarel) [reply data/items.js]: This should probably just be changed back.

Comment (KrisXV) [reply data/items.js]: You’re also not supposed to use accented e because smogon standard is to not use them

Comment (Zarel) [reply data/items.js]: Do we need to follow Smogon standard?

File: data/items.js [hunk 2]
```diff
@@ -1705,7 +1705,7 @@ let BattleItems = {
 		},
 		num: 34,
 		gen: 2,
-		desc: "Restores 50 HP to one Pokemon but lowers Happiness.",
+		desc: "Restores 60 HP to one Pokemon",
```
Comment (KrisXV) [inline data/items.js]: This is missing a period

**Comment by YaBoiJD** [comment]
@Zarel, Should I make a better description for the Utility umbrella?

**Comment by Zarel** [comment]
If you want, sure.

**Comment by KrisXV** [comment]
the shortdesc of Utility Umbrella should just be `Blocks the holder from the effects of sun/rain`, but the normal description should go into all of the details

**Comment by YaBoiJD** [comment]
@Zarel should this be closed, or should I leave open?

**Pull request merged**